modules/webgl/src/adapter/webgl-device.ts

```diff
@@ -412,6 +416,10 @@ ${device.info.vendor}, ${device.info.renderer} for canvas: ${device.canvasContex
     clear(this, options);
   }
 
+  override resetWebGL(): void {
+    resetGLParameters(this.gl);
```

Comment:
`log.warn` to briefly say it is for debug only, to avoid it creeping into production apps?